packages/babel-types/src/clone/cloneNode.ts

```diff
@@ -31,10 +30,16 @@ export default function cloneNode<T extends t.Node>(
   deep: boolean = true,
   withoutLoc: boolean = false,
 ): T {
-  if (!node) return node;
+  return cloneNodeInternal(node, deep, withoutLoc, new Map());
+}
 
-  const isTop = !commentsCache.get("inCloning");
-  if (isTop) commentsCache.set("inCloning", true);
+function cloneNodeInternal<T extends t.Node>(
+  node: T,
+  deep: boolean = true,
+  withoutLoc: boolean = false,
+  commentsCache: Map<t.Comment, t.Comment> | null,
```

Comment:
When will it be `null`?

Reply:
Well, `cloneIfNodeOrArray` is not exported :sweat_smile:

Reply:
How are you using `Object.keys`? `module.exports` has a single property, which is `cloneNode`. It's impossible to access `cloneIfNodeOrArray` from this file :thinking:

Reply:
I tested it and you are right!
May be it was available in a few years old version.
I will remove this compatibility.